app/helpers/proxmox_vm_os_template_helper.rb

```diff
@@ -41,7 +41,6 @@ def parse_container_ostemplate(args)
     ostemplate = args['ostemplate']
     ostemplate_file = args['ostemplate_file']
     ostemplate ||= ostemplate_file
```

Comment:
You could even simplify this to
```suggestion
    ostemplate = args['ostemplate'] || args['ostemplate_file']
```